afs/media/js/views/components/reports/observation.js

```diff
@@ -34,9 +34,14 @@ define([
                 'subjectOf': undefined,
                 'label': undefined
             };
+            self.substanceDataConfig = {
+                dimension: undefined,
+                timespan: {nodegroup: 'timespan', key: 'timespan of observation'}
```

Comment:
I'd rename 'nodegroup' to 'path' since that's how we're referring to this in scenes.

Reply:
Renamed